### PR TITLE
chore: initial setup

### PR DESCRIPTION
**Pull request by DorisWongNHS**

### CODE REVIEW

File: .aws/task-definition-template.json [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
+          "awslogs-group": "awslogs-${environment}-tis-trainee-credentials",
           "awslogs-region": "eu-west-2",
-          "awslogs-stream-prefix": "awslogs-tis-template"
+          "awslogs-stream-prefix": "awslogs-tis-trainee-credentials"
         }
       },
       "portMappings": [
         {
-          "containerPort": 8080
+          "containerPort": 8209
```
Comment (Judge40) [inline .aws/task-definition-template.json:22]: 8209 is taken by the NDW exporter, I set the infra up to use 8210

File: README.md [hunk 2]
```diff
@@ -1,49 +1,54 @@
...
- - Update copyright year in [TemplateApplication].
- - Update copyright year in [TemplateApplicationTest].
- - Update this README.
- - Sentry:
-    - Set up Sentry project.
-    - Provide `SENTRY_DSN` and `SENTRY_ENVIRONMENT` as environmental variables
-   during deployment.
  - Sonar:
     - Add repository to SonarCloud.
     - Add SonarCloud API key to repository secrets.
```
Comment (Judge40) [inline README.md:9]: This is already done

**Review by Judge40 — CHANGES_REQUESTED** [review]
`sonar.projectKey` project key is available as `Health-Education-England_tis-trainee-credentials` so could be set in the build file.

File: .aws/task-definition-template.json (hunk 1 above)
Comment (DorisWongNHS) [reply .aws/task-definition-template.json]: It is updated to 8210

**Review by Judge40 — APPROVED** [review]

**Pull request merged**